syntax = "proto3";

package model;

option go_package = "/modelpb";


service GoService {
  rpc SendTrainingData (TrainingRequest) returns (TrainingResponse);
}

message TrainingRequest {
  repeated string feature_names = 1;
  repeated float values = 2;
}

message TrainingResponse {
  string status = 1;
}
